pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -64,13 +66,15 @@ const (
 	// Formerly, the `500 * 1024 / 450` expression was used instead.
 	metricsAlertsLinesLimit = 1000
 
+	// InstallPlansTopX is the Maximal number of Install plans by non-unique instances count
+	InstallPlansTopX = 100
```

Comment:
I guess there's something worng with this limit. I tried to set it to 5 and got `{"items":[],"stats":{"TOTAL_COUNT":9,"TOTAL_NONUNIQ_COUNT":9}}` which means there was 9 installplans but can't see the five in the items array.